src/components/Cards/Base.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactNode } from "react";
+import styled from "@emotion/styled";
+import colors from "../../styles/colors";
+
+const CARD_MAX_WIDTH = "544px";
+const CARD_BORDER_RADIUS = "5px";
+const CARD_BORDER_COLOR = colors.lightgrey;
+const SMALL_SCREEN_BREAKPOINT = "576px";
+const IMAGE_CONTAINER_MIN_HEIGHT = "135px";
+
+const CardContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: ${CARD_MAX_WIDTH};
+
+  /* desktop styles */
+  @media (min-width: ${SMALL_SCREEN_BREAKPOINT}) {
+    flex-direction: row;
+  }
+`;
+
+type CardImageContainerProps = {
+  imgSrc: string;
+};
+
+const CardImageContainer = styled.div`
+  /* position relative so that the overlay text can be absolutely positioned */
+  position: relative;
+  /* set min-height for image to prevent div collapsing in small screens */
+  min-height: ${IMAGE_CONTAINER_MIN_HEIGHT};
+  border-radius: ${CARD_BORDER_RADIUS} ${CARD_BORDER_RADIUS} 0 0;
+  background-image: linear-gradient(
+      transparent 50%,
+      rgba(51, 51, 51, 0.1) 60%,
+      rgba(51, 51, 51, 0.3) 70%,
+      rgba(51, 51, 51, 0.5) 80%,
+      rgba(51, 51, 51, 0.8) 100%
```

Comment:
If I'm understanding this correctly this makes container gradually darker to ensure there's enough contrast between the white date text and the background? Is the white date the source of the accessibility 'incomplete' warning on storybook?

Reply:
That's correct. This adds the gradient. Also correct that it is the source of the accessibility incomplete warning. Any thoughts on how to address this warning?
![Screen Shot 2020-09-19 at 3 50 09 PM](https://user-images.githubusercontent.com/13170966/93690661-e9eb4a00-fa8f-11ea-9db0-d1c1267c234d.png)

Reply:
IIRC, you can have both a `background-color` and a `background-image` property on an element and `background-color` is "underneath" the image. In this case it may be good to just default to something that we know will contrast from the text as the color as a safety / default?

Reply:
If we were to add a background-color, it would have to be in the `span` element where the violation occurs (line 54-61 of MeetingCard.tsx). Because if we had a background-color to `CardImageContainer` the violation would still occur. Not sure what to do.

Reply:
I think because it is "incomplete" and not a "violation" I am okay with merging it in. We can think about how to solve it later.

Reply:
If we come up with a better solution that is. 🤷